<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>The Judge v0.0.1</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="loadImagesTab">
       <attribute name="title">
        <string>Load images</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item alignment="Qt::AlignTop">
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="addImagesButton">
                  <property name="text">
                   <string>Add image(s)</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="removeImagesButton">
                  <property name="text">
                   <string>Remove selected image(s)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="imageListWidget">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::MultiSelection</enum>
            </property>
            <property name="selectionRectVisible">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignRight|Qt::AlignVCenter">
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>6</number>
         </property>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="markButton">
         <property name="text">
          <string>Mark</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveResultsButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save results</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>542</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_exam"/>
    <addaction name="actionSave_results"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionLoad_exam">
   <property name="text">
    <string>Load exam</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionSave_results">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save results</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionLoad_exam</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadExamFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>quit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addImagesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>loadExamImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeImagesButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>removeExamImages()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>177</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>markButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>mark()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>imageListWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>imageListItemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>269</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_results</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveExamResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveResultsButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveExamResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>495</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>quit()</slot>
  <slot>loadExamFile()</slot>
  <slot>mark()</slot>
  <slot>loadExamImages()</slot>
  <slot>removeExamImages()</slot>
  <slot>imageListItemSelectionChanged()</slot>
  <slot>saveExamResults()</slot>
 </slots>
</ui>
